packages/amplify-graphql-api-construct-tests/src/__tests__/sql-models.test.ts

```diff
@@ -199,7 +300,23 @@ const cleanupDatabase = async (options: { identifier: string; region: string; db
  * @param dbDetails the details object
  * @param projRoot the destination directory to write the `db-details.json` file to
  */
-const writeDbDetails = (dbDetails: DBDetails, projRoot: string): void => {
+const writeDbDetails = (
+  dbDetails: {
```

Comment:
Suggest we reuse the `DBDetails` type instead of manually specifying the shape

```ts
dbDetails: DBDetails & Omit<DBDetails, 'connectionConfigs'> & {dbConnectionConfig: SqlModelDataSourceDbConnectionConfig}
```

Alternately we could declare a base shape & extend it for individual DB connections vs. the aggregate connection.